sanity/doctors/EstateDoctor.js

```diff
@@ -44,41 +44,57 @@ export class EstateDoctor extends Doctor {
     const {
       id,
       owner,
+      operator,
+      update_operator,
       data: { parcels }
```

Comment:
I know this is not from this PR but the rest of the codebase tends to steer away from this double destructuring. I looked around and it's the only place I found where we're using it. If you don't mind, can we split it into two lines?